mne/decoding/tests/test_time_gen.py

```diff
@@ -223,3 +229,22 @@ def predict(self, x):
                                                    scorer=scorer)
                     gat.fit(epochs, y=y_)
                     gat.score(epochs, y=y_)
+
+
+@slow_test
```

Comment:
how long does the test take? any way you can speed it up? Maybe use fewer features?